templates/authors/author_page.html

```diff
@@ -0,0 +1,77 @@
+{% extends 'base_page.html' %}
+
+{% load static %}
+{% load wagtailcore_tags %}
+{% load wagtailimages_tags %}
+{% block breadcrumb %}{% endblock %}
+{% block content %}
+    {% include 'includes/breadcrumb.html' %}
+    <div class="author-intro">
+        <div class="container">
+            <div class="row">
+                <div class="col-sm-12 col-md-3 order-md-2">
+                    {% image page.image fill-500x500 as author_img %}
+                    <img src="{{ author_img.url }}" alt="{{ author_img.alt }}" class="author-intro__image">
+                </div>
+
+                <div class="col-sm-12 col-md-9 order-md-1 info">
+                    <h1 class="author-intro__title">
+                        {{ page.title }}
+                    </h1>
+                    {% if page.role %}
+                    <div class="author-intro__paragraph">
+                        {{ page.role|richtext }}
+                    </div>
+                    {% endif %}
+                </div>
+            </div>
+
+
+<!--            <div class="row">-->
+<!--                <div class="col-9 info">-->
+<!--                    <h1 class="author-intro__title">-->
+<!--                        {{ page.title }}-->
+<!--                    </h1>-->
+<!--                    {% if page.role %}-->
+<!--                        <div class="author-intro__paragraph">-->
+<!--                            {{ page.role|richtext }}-->
+<!--                        </div>-->
+<!--                    {% endif %}-->
+<!--                </div>-->
+<!--                <div class="col-3">-->
+<!--                    {% image page.image fill-500x500 as author_img %}-->
+<!--                    <img src="{{ author_img.url }}" alt="{{ author_img.alt }}" class="author-intro__image">-->
+<!--                </div>-->
+<!--            </div>-->
+<!--            -->
+        </div>
+    </div>
+
+
+    
+    <div class="article-container container">
+        <div class="article-container__main">
+            <h2 class="related-content__heading">About the author</h2>
+            {{ page.summary | richtext }}
+        </div>
+    </div>
+
+    <div class="related-content related-content--borderless">
+        <div class="container">
+            {% if page.authored_focused_articles %}
+                <h2 class="related-content__heading">{{page.title}}'s articles and blogs</h2>
+
+                <ul class="card-group--list-style-none">
+                    {% for item in page.authored_focused_articles %}
+                        {% include 'includes/card-group-secondary-nav.html' with link_page=item heading="More stories" card_type="Featured" show_type_label=True %}
+                    {% endfor %}
+                </ul>
+            {% endif %}
+        </div>
+    </div>
+
+{% endblock %}
+
+{% block extra_js %}
+    <script src="{% static 'scripts/article.js' %}"></script>
```

Comment:
Is there any associated JavaScript with this?

Reply:
@ahosgood, I'm not entirely clear what <script src="{% static 'scripts/article.js' %}"></script> is doing here - it's quite a large/complex file, might be safest to keep it in place for now?